lib/dependencies/DepBlockHelpers.js

```diff
@@ -33,7 +33,7 @@ DepBlockHelpers.getDepBlockPromise = function(depBlock, outputOptions, requestSh
 				"])";
 		}
 	}
-	return "Promise.resolve()";
+	return "new Promise(function(resolve) { resolve(); })";
```

Comment:
Perhaps this could be parametrized by a webpack configuration option (using `.resolve` by default)? If `Promise.resolve()` is in fact faster (didn't test), it's also quite a bit shorter, then it might make sense.